reference-implementation/lib/readable-stream.js

```diff
@@ -251,6 +257,33 @@ class ReadableStream {
     const branches = ReadableStreamTee(this, false);
     return createArrayFromList(branches);
   }
+
+  [InternalTranfer](/* targetRealm */) {
```

Comment:
This would allow you to write web platform tests using iframes (although they might not do so great in our test runner... but maybe they would!)